README.md

```diff
@@ -56,16 +58,15 @@ See [the latest GitHub release](https://github.com/pingidentity/pingcli/releases
 OR
 
 Use the following single-line powershell command to install Ping CLI into '%LOCALAPPDATA%\Programs' directly.
-```text
-$latestReleaseUrl = Invoke-WebRequest -Uri "https://github.com/pingidentity/pingcli/releases/latest" -MaximumRedirection 0 -ErrorAction Ignore -UseBasicParsing -SkipHttpErrorCheck; `
+```powershell
+$latestReleaseUrl = Invoke-WebRequest -Uri "https://github.com/pingidentity/pingcli/releases/latest" -MaximumRedirection 0 -ErrorAction Ignore -UseBasicParsing; `
 $RELEASE_VERSION = [System.IO.Path]::GetFileName($latestReleaseUrl.Headers.Location); `
 $RELEASE_VERSION_NO_PREFIX = $RELEASE_VERSION -replace "^v", ""; `
-$uname = (uname -m); `
-$HARDWARE_PLATFORM = $uname -replace "aarch64", "arm64" -replace "x86_64", "amd64"; `
+$HARDWARE_PLATFORM = $env:PROCESSOR_ARCHITECTURE -replace "ARM", "arm64" -replace "ARM64", "arm64" -replace "x86", "amd64"; `
```

Comment:
It looks like possible values here are `AMD64`, `IA64`, `ARM64`, `EM64T`, and `X86`